process/smartContract/hooks/blockChainHook.go

```diff
@@ -272,6 +279,19 @@ func (bh *BlockChainHookImpl) GetStorageData(accountAddress []byte, index []byte
 	return value, trieDepth, nil
 }
 
+func (bh *BlockChainHookImpl) syncMissingDataTrieNode(err error) {
```

Comment:
rename this function?
checkAndSyncMissingTrieNode?

Reply:
Renamed to `sync**If**MissingDataTrieNode`